easybuild/easyconfigs/g/GSL/GSL-2.2.1-CrayIntel-2016.11.eb

```diff
@@ -0,0 +1,51 @@
+# Built with EasyBuild version 3.0.1 on 2017-01-20_18-49-28
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = "2.2.1"
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+ The library provides a wide range of mathematical routines such as random number generators, special functions
+ and least-squares fitting."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2016.11'}
+toolchainopts = {'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = "--with-pic"
+
+sanity_check_paths={
+    'files': ['lib/libgsl.so', 'lib/libgsl.a'],
+    'dirs': ['include'],
+}
+
+moduleclass = 'numlib'
+
+# Build statistics
```

Comment:
Please remove the buildstats from here (it makes more sense to keep it on the ebrepo directory rather than here).